assets/shaders/RayTracing.rgen

```diff
@@ -83,6 +83,8 @@ void main()
 
 	const bool accumulate = Camera.NumberOfSamples != Camera.TotalNumberOfSamples;
 	const vec3 accumulatedColor = (accumulate ? imageLoad(AccumulationImage, ivec2(gl_LaunchIDEXT.xy)) : vec4(0)).rgb + pixelColor;
+	
+	imageStore(AccumulationImage, ivec2(gl_LaunchIDEXT.xy), vec4(accumulatedColor, 0));
```

Comment:
I would reasonably expect this optimisation ought to be done by the driver. Unless Vulkan specifically mandates that the image store act as a barrier? (I doubt). 

I definitely want to test this at some point in the 2080 Ti and 3090.

Reply:
This actually gets optimized away in wave64 mode. I assume there is currently a bug in wave32 mode that causes it to not be optimized.

Reply:
CapFrameX on Twitter already tested it on Turing, and the result was exactly as expected. There was no impact at all.